packages/api-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api-sdk",
-  "version": "7.18.0",
+  "version": "7.18.2-alpha.0",
```

Comment:
The changes we're introducing are breaking changes, right? That should result in a major version bump.